extension/content.css

```diff
@@ -94,43 +94,38 @@ readerease-sentence {
 #readingDisplayContainer {
```

Comment:
These are qualitative, aesthetic changes and thus will be hard to examine from code; checkout branch for demo, also will post pic in Slack dev channel